src/components/format-date/format-date.test.ts

```diff
@@ -53,10 +53,11 @@ describe('<sl-format-date>', () => {
       it(`date has correct language format: ${setup.lang}`, async () => {
         const el = await fixture<SlFormatDate>(
           html`
-            <sl-format-date .date="${new Date(new Date().getFullYear(), 0, 1)}" lang="${setup.lang}"></sl-format-date>
```

Comment:
These were set as properties so the date object would be passed. We can also test string parsing with ISO strings in additional tests, which are documented as the suggested format when passing dates as an attribute.

Reply:
e.g. `<sl-format-date date="${new Date().toISOString()`}">`